app/src/main/java/pl/futurecollars/invoicing/model/Vat.java

```diff
@@ -0,0 +1,17 @@
+package pl.futurecollars.invoicing.model;
+
+import java.math.BigDecimal;
+
+public enum Vat {
+    V_23(new BigDecimal("0.23")),
```

Comment:
You need to put empty line here. 
Error: eckstyle] [ERROR] /home/runner/work/invoicing-system-Pawel-Klimek/invoicing-system-Pawel-Klimek/app/src/main/java/pl/futurecollars/invoicing/model/Vat.java:5: Leave empty row after class/interface/enum definition! [RegexpMultiline]

Reply:
And you have a lot of other checkstyle violations:
  Checkstyle files with violations: 8
  Checkstyle violations by severity: [warning:44]